### ISSUE TITLE
Should search well-known locations for Elasticsearch binary

### ISSUE THREAD

**Issue by georgevreilly** [jong/testing.elasticsearch#5]
You rely on the Elasticsearch binary to be on the path. Your ancestor, testing.postgresql, has a notion of [SEARCH_PATHS](https://github.com/tk0miya/testing.postgresql/blob/master/src/testing/postgresql.py#L32).

Likely values:
- `/usr/local/bin`  # Mac homebrew
- `/usr/share/elasticsearch/bin`  # RHEL (Bamboo); Ubuntu Trusty (ELS instances, inside Ubuntu Docker containers)

**Comment by jong** [comment]
I agree, this would be a nice addition. Honestly when implementing this I just knew my use case would already have the binary in the PATH and it didn't seem worth implementing functionality that I wasn't going to immediately use.
